### ISSUE TITLE
Docs page hard to find from main site

### ISSUE THREAD

**Issue by tringenbach** [usebruno/bruno-website#39]
I think the Docs site should be in the banner at the top of the main website. 

More than once I wasted a lot of time trying to find the link to the docs site. When I first started lookin gat Bruno, I was actually concerned that there wasn't documentation! I'm glad I was wrong about that. 

These days I just google `bruno docs`. But I figure, it was confusing me, it's probably confusing other people, and so you might want to know and consider changing it. Thanks!

**Comment by ethancane** [comment]
Agreed. Just came across this project and already I paid upfront for the Golden Edition to support the project. Definitely would like to see a more prominent link to documentation in the main header navigation.
